client/components/common/BaseForm.vue

```diff
@@ -108,4 +114,42 @@ span {
   display: flex;
   align-items: center;
 }
+
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltiptext {
+  visibility: hidden;
+  background-color: black;
+  color: #fff;
+  text-align: center;
+  padding: 5px 0;
+  border-radius: 6px;
+ 
+  /* Position the tooltip text - see examples below! */
+  position: absolute;
+  z-index: 1;
+  width: 500px;
```

Comment:
I would rather not have a fixed width. In general there is a lot hardcoded here which I'm concerned about.